File: easy/448 - find-all-numbers-disappeared-in-an-array/Solution.py
class Solution(object):
    def findDisappearedNumbers(self, n):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        s=[]
        for i in range(1,len(n)+1):
            if i not in n :
                s.append(i)
        return s

File: easy/448 - find-all-numbers-disappeared-in-an-array/Solution.cpp
class Solution {
public:
    vector<int> findDisappearedNumbers(vector<int>& n) {
       vector<int>v;
       sort(n.begin(),n.end());
       for(int i=0,s=1;i<n.size();i++,s++){
        if(s!=n[i])
            v.push_back(s);
       }
       return v;
    }
};